linkage-monitor/src/test/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitorTest.java

```diff
@@ -180,22 +183,29 @@ public void testBomSnapshot()
   @Test
   public void generateMessageForNewError() {
     Set<SymbolProblem> baselineProblems = ImmutableSet.of(classNotFoundProblem);
+    Path jar = Paths.get("aaa", "ccc-1.2.3.jar");
     ImmutableSetMultimap<SymbolProblem, ClassFile> snapshotProblems =
         ImmutableSetMultimap.of(
             classNotFoundProblem, // This is in baseline. It should not be printed
-            new ClassFile(Paths.get("aaa", "bbb-1.2.3.jar"), "com.abc.AAA"),
+            new ClassFile(jar, "com.abc.AAA"),
             methodNotFoundProblem,
-            new ClassFile(Paths.get("aaa", "bbb-1.2.3.jar"), "com.abc.AAA"),
+            new ClassFile(jar, "com.abc.AAA"),
             methodNotFoundProblem,
-            new ClassFile(Paths.get("aaa", "bbb-1.2.3.jar"), "com.abc.BBB"));
+            new ClassFile(jar, "com.abc.BBB"));
 
-    String message = LinkageMonitor.messageForNewErrors(snapshotProblems, baselineProblems);
+    DependencyPath dependencyPath = new DependencyPath();
+    dependencyPath.add(new DefaultArtifact("foo:bar:1.0.0"));
+    dependencyPath.add(new DefaultArtifact("aaa:ccc:1.2.3"));
+    String message =
+        LinkageMonitor.messageForNewErrors(
+            snapshotProblems, baselineProblems, ImmutableListMultimap.of(jar, dependencyPath));
     assertEquals(
         "Newly introduced problem:\n"
             + "(bbb-1.2.3.jar) io.grpc.protobuf.ProtoUtils.marshaller's method"
```

Comment:
is the method name (here "marshaller") being listed twice?

Reply:
Good catch.

Fixed the dummy data.